.editorconfig

```diff
@@ -0,0 +1,26 @@
+root = true
+
+[*]
+trim_trailing_whitespace = true
+end_of_line = lf
+
+[*.py]
+indent_style = space
+indent_size = 4
+charset = utf-8
+insert_final_newline = true
```

Comment:
Are there file types for which we wouldn't want these charset and insert_final_newline settings, or could this be in the \[\*\] block above instead of copied in each of these filetype blocks?

Reply:
I cmmon rules combined into one.